dist/windows/installer-translations/english.nsi

```diff
@@ -30,6 +30,8 @@ LangString launch_qbt ${LANG_ENGLISH} "Launch qBittorrent."
 LangString inst_requires_64bit ${LANG_ENGLISH} "This installer works only in 64-bit Windows versions."
 ;LangString inst_requires_win7 ${LANG_ENGLISH} "This qBittorrent version requires at least Windows 7."
 LangString inst_requires_win7 ${LANG_ENGLISH} "This qBittorrent version requires at least Windows 7."
+;LangString inst_uninstall_link_description ${LANG_ENGLISH} "Uninstall qBittorrent"
+LangString inst_uninstall_link_description ${LANG_ENGLISH} "Uninstall qBittorrent"
```

Comment:
@bovirus 
Maybe it is better to copy this new string to all other language files? For example PR #14697 won't know this new string existence.

Reply:
@Chocobo1 

I believe that it's better that you made all changes copying this string in all languages and the I re-translate in italian language.

I prefer that because I hope to avoid to make some erros in many files.